CRM/Admin/Form/ScheduleReminders.php

```diff
@@ -46,6 +46,15 @@ public function setComponentID($compId): void {
     $this->_compId = $compId;
   }
 
+  /**
+   * Build all the data structures needed to build the form.
+   */
+  public function preProcess() {
+    parent::preProcess();
+
+    $this->_id = CRM_Utils_Request::retrieve('id', 'Positive');
+  }
```

Comment:
@chrisgaraffa I think is not necessary to override the preprocess to assign the `$this->_id` because the parent class already does that [here](https://github.com/civicrm/civicrm-core/blob/master/CRM/Admin/Form.php#L63). Or is it not working earlier?